src/components/film-detail/film-detail.component.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import {apiBaseUrl} from '../../infrastructure';
+import './film-detail.component.scss';
+import {withRouter, Link} from 'react-router-dom';
+
+class FilmDetailComponent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            film: {}
+        }
+    }
+
+    componentDidMount() {
```

Comment:
this belongs to the container
also you already have film request in the container